workflows/ts2fs.py

```diff
@@ -288,7 +363,9 @@ def _generate_grapes_fs(neu_fs, nonneu_fs, output, is_folded, seq_len, sample_si
         nonneu_prop float: Proportion of non-neutral mutations
     """
     neu_len = round(seq_len * kwargs['neu_prop'])
+    neu_fs[0] = neu_len - sum(neu_fs[1:]) 
     nonneu_len = round(seq_len * kwargs['nonneu_prop'])
+    nonneu_fs[0] = nonneu_len - sum(nonneu_fs[1:]) 
 
```

Comment:
Need to add something similar in `_generate_dfe_alpha_fs`. Right now the first bin is ALL positions including mutated and substitutions. This is the right format form some software, but not for DFE-alpha.